### PR TITLE
Pub upgrade

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
CC @alorenzen

File: mono_repo/lib/src/pub_command.dart [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import 'package:path/path.dart' as p;
+
+import 'utils.dart';
+
+class PubCommand extends Command {
+  @override
+  String get name => 'pub';
+
+  @override
+  String get description => 'Run `pub upgrade` against all packages.';
```
Comment (alorenzen) [inline mono_repo/lib/src/pub_command.dart]: Should we add a flag for pub get vs. pub upgrade? Or in general do we also encourage folks to just run pub upgrade?

File: mono_repo/lib/src/pub_command.dart [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+import 'package:args/command_runner.dart';
+import 'package:io/ansi.dart';
+import 'package:io/io.dart';
+
+import 'package:path/path.dart' as p;
+
+import 'utils.dart';
+
+class PubCommand extends Command {
```
Comment (jakemac53) [inline mono_repo/lib/src/pub_command.dart]: `PubUpgradeCommand` (and similar changes to other variables/fields below)?

Or, just generalize this and leave it as `PubCommand` by just passing through all extra arguments to `pub`. That might actually be my preference?

File: mono_repo/lib/src/pub_command.dart [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    // TODO(kevmoo): https://github.com/dart-lang/io/issues/22
+    Directory.current = workingDir;
+
+    var proc = await processManager.spawn('pub', arguments: ['upgrade']);
+
+    var exit = await proc.exitCode;
+
+    if (exit == 0) {
+      print(wrapWith('`$dir`: success!', [styleBold, lightBlue]));
```
Comment (jakemac53) [inline mono_repo/lib/src/pub_command.dart]: green for consistency w/presubmit?

Comment (kevmoo) [reply mono_repo/lib/src/pub_command.dart]: yup

File: mono_repo/lib/src/pub_command.dart (hunk 2 above)
Comment (kevmoo) [reply mono_repo/lib/src/pub_command.dart]: yup

File: mono_repo/lib/src/pub_command.dart (hunk 1 above)
Comment (kevmoo) [reply mono_repo/lib/src/pub_command.dart]: yup

**Comment by kevmoo** [comment]
PTAL

**Comment by jakemac53** [comment]
Test might be nice but not strictly necessary imo

**Comment by kevmoo** [comment]
Agreed :-)

**Pull request merged**